VAMobile/src/api/appointments/getAppointments.tsx

```diff
@@ -36,8 +39,13 @@ export const useAppointments = (
   page: number,
   options?: { enabled?: boolean },
 ) => {
+  const { data: authorizedServices } = useAuthorizedServices()
+  const appointmentsInDowntime = useDowntime('appointments')
+  const queryEnabled = options && has(options, 'enabled') ? options.enabled : true
```

Comment:
This conditional looks funky, but we need to check whether the `options` object has the `enabled` property instead of just checking `options?.enabled`. Mainly because when the `enabled` option isn't set on the screen that's using this hook, its value will result in `undefined`, which would make this conditional `false`, but we want it to run by default if the `enabled` option isn't specified (assuming the feature isn't in maintenance and the user have access to the feature).

I was also considering turning this into a helper function that all queries can reference, but I manually added it to the queries for now since the logic is fairly simple, and we're only using this in a handful of queries.